src/Raven.Server/Documents/Indexes/Static/JavaScriptMapOperation.cs

```diff
@@ -68,9 +66,9 @@ public IEnumerable IndexingFunction(IEnumerable<object> items)
                         if (jsItem.IsArray())
                         {
                             var array = jsItem.AsArray();
-                            foreach (var (prop, val) in array.GetOwnPropertiesWithoutLength())
+                            foreach (var val in array)
```

Comment:
this is without `.length` now?

Reply:
Seems that resolving didn't attach my comments. Writing them again...

Yes the enumerator for `JsArray` works like what you would expect from regular .NET array. So only values are returned between 0...length. It's also optimized not to do any unnecessary `JsString` conversions for keys etc. Suited for your use case I'd say.

We also have a test bed for RavenDB that contains test cases that we don't accidentally want to break: https://github.com/sebastienros/jint/blob/main/Jint.Tests.PublicInterface/RavenApiUsageTests.cs , all additions are welcome there.